metatomic-torch/CMakeLists.txt

```diff
@@ -195,15 +193,7 @@ FetchContent_Declare(nlohmann_json
     URL https://github.com/nlohmann/json/releases/download/v3.11.3/json.tar.xz
     URL_HASH SHA256=d6c65aca6b1ed68e7a182f4757257b107ae403032760ed6ef121c9d55e81757d
 )
-if (CMAKE_VERSION VERSION_GREATER 3.18)
-    FetchContent_MakeAvailable(nlohmann_json)
-else()
-    if (NOT nlohmann_json_POPULATED)
-        FetchContent_Populate(nlohmann_json)
-    endif()
-
-    add_subdirectory(${nlohmann_json_SOURCE_DIR} ${nlohmann_json_BINARY_DIR})
-endif()
+FetchContent_MakeAvailable(nlohmann_json)
```

Comment:
Actually not sure we can assume `3.18` for everyone...

Reply:
no, we ask for 3.16, which is the one in ubuntu 20.04

The code previously handled stuff differently for cmake < 3.18 and cmake >=3.18, but this was actually not required